docs/source/Tutorial.md

```diff
@@ -1575,6 +1570,10 @@ a bit more flexibility.
   monster_builder.add_equipped(axe.Union());
   auto orc = monster_builder.Finish();
 ~~~
+**Important**: Unlike structs, you should not interlace creation of tables or other objects,
+which is why we created all the strings/vectors/tables that this monster refers
+to beforehand. If you try to create any of them after creation of `monster_builder` and before the corresponding call to `Finish()`,
+you will get an exception.
```

Comment:
In C++, you will get an `assert`, it doesn't use exceptions anywhere in the code. Keep the original comment?

Reply:
Oh, good to know, thanks!